<?xml version="1.0"?>
<launch>
  <arg name="model_path" default="yolov8n.pt" />
  <arg name="confidence_threshold" default="0.5" />
  <arg name="image_topic" default="/camera/color/image_raw" />
  
  <node name="yolo_detector" pkg="perception_yolo" type="yolo_detector_node.py" output="screen">
    <param name="model_path" value="$(arg model_path)" />
    <param name="confidence_threshold" value="$(arg confidence_threshold)" />
    <param name="image_topic" value="$(arg image_topic)" />
  </node>
</launch>
